lib/router/mbaas.js

```diff
@@ -6,10 +6,10 @@ var userAuth = require('./mbaas-auth');
 var sessionMiddleware = require('./mbaas-session-middleware');
 var _ = require('lodash');
 
-function initRouter(mediator, authResponseExclusionList) {
+function initRouter(mediator, authResponseExclusionList, expressSessionMiddleware) {
   var router = express.Router();
 
-  router.all('/auth', function(req, res) {
+  router.all('/auth', function(req, res, next) {
```

Comment:
I removed the `next` function. Including it in the route as you showed above would execute it for every authentication request.